corehq/apps/data_analytics/malt_generator.py

```diff
@@ -0,0 +1,75 @@
+from django.db import transaction
+
+from corehq.apps.app_manager.models import Application
+from corehq.apps.data_analytics.models import MALTRow
+from corehq.apps.domain.models import Domain
+from corehq.apps.smsforms.app import COMMCONNECT_DEVICE_ID
+from corehq.apps.sofabed.models import FormData
+
+
+class MALTTableGenerator(object):
+    """
+        Populates SQL table with data for given datespan
+        See MALTRow
+    """
+    _wam_pam_cache = {}
+
+    def __init__(self, datespan_object):
+        self.datespan = datespan_object
+
+    @transaction.atomic
+    def build_table(self):
+
+        for domain in Domain.get_all():
+            for user in domain.all_users():
+                forms_query = self.get_forms_queryset(user._id, domain.name)
```

Comment:
Yeah, these are spot on, but as cory and I decided that we should let this be as is (performance wise) and see how this runs, I stopped thinking at alternatives. 

But I am going to write the bulk-create instead of one per each. I thought `transaction.atomic` takes care of batch update, but I am removing that. So I will add a bulk_create.